bundles/binding/org.openhab.binding.comfoair/README.md

```diff
@@ -1,458 +1,1007 @@
-# ComfoAir Binding
-
-This binding should be compatible with the Zehnder ComfoAir 350 ventilation system. ComfoAir 550 is untested but should supposedly use the same protocol. The same is true for the device WHR930 of StorkAir, G90-380 by Wernig and Santos 370 DC to Paul.
-
-## Binding Configuration
-
-You can configure this binding in the file `services/comfoair.cfg`.
-
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| port     |         |   Yes    | Serial port which is connected to the Zehnder ComfoAir system, for example `/dev/ttyS0` on Linux or `COM1` on Windows |
-| refresh  | 60000   |   No     | refresh inverval in milliseconds
-
-
-### Example
-
```

Comment:
Headers should be followed by a blank line.

Reply:
Done.